src/entities/Profile.ts

```diff
@@ -43,12 +43,15 @@ export default class Profile {
 	public avatar!: boolean;
 
 	@Column({ nullable: true })
+	@Matches(/^(?!.*\.\.)(?!.*\.$)[^\W][\w.]{0,29}$/, { message: 'Not a valid instagram username' })
 	public instagram?: string;
 
 	@Column({ nullable: true })
+	@Matches(/^[A-Za-z0-9\.]{5,}$/, { message: 'Not a valid facebook username' })
```

Comment:
Just a note, but you could simplify this to `/^[a-z0-9\.]{5,}$/i` - the `i` flag at the end means case-insensitive!